actions/post_actions.jsx

```diff
@@ -203,12 +204,8 @@ export function emitEmojiPosted(emoji) {
 const POST_INCREASE_AMOUNT = Constants.POST_CHUNK_SIZE / 2;
 
 // Returns true if there are more posts to load
-export function increasePostVisibility(channelId, focusedPostId) {
+export function loadPosts({channelId, postId, type}) {
     return async (doDispatch, doGetState) => {
-        if (doGetState().views.channel.loadingPosts[channelId]) {
-            return true;
-        }
-
         const currentPostVisibility = doGetState().views.channel.postVisibility[channelId];
```

Comment:
Are there any changes that need to be made to how `postVisibility` works now?

Reply:
As of now, it works. This might be removed for virtualization soon. Even without it, I think we can remove and work with filtering from react.